make/build.js

```diff
@@ -700,7 +700,7 @@ module.exports = function build(settings, task) {
         this.setting("closure_version");
         var opts = {
             language_in: "ECMASCRIPT_2018",
-            language_out: "ECMASCRIPT_2018",
+            language_out: "ECMASCRIPT_2016",
```

Comment:
Does it make sense to use `ECMASCRIPT_2016` here (it was `ECMASCRIPT_2018`) – the CindyXR plugin needs WebXR, and I do not think that any browser capable of doing WebXR is not capable of doing `ECMASCRIPT_2018`…